srcpkgs/pantheon-default-settings/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'pantheon-default-settings'
+pkgname=pantheon-default-settings
+version=5.1.2
+revision=1
+wrksrc="default-settings-${version}"
+depends="io.elementary.icons io.elementary.stylesheet fonts-droid-ttf"
+short_desc="Default settings for elementary OS"
+maintainer="linarcx <linarcx@riseup.net>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/elementary/default-settings"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=7e488ed4c3599af574e4337b0ed0da68246ac5e65b0b022d72ed569415feebfa
+
+do_install() {
+	mkdir -p ${PKGDESTDIR}/etc/skel/.config
+	vinstall overrides/default-settings.gschema.override 644 /usr/share/glib-2.0/schemas/25_pantheon-default-settings.gschema.override
```

Comment:
this is wrong, it should be

```
vinstall overrides/default-settings.gschema.override 644 /usr/share/glib-2.0/schemas 25_pantheon-default-settings.gschema.override
```
4 arguments

Reply:
Done!